src/client/tile.cpp

```diff
@@ -500,7 +500,7 @@ u32 TextureSource::getTextureId(const std::string &name)
 		while(true) {
 			// Wait result for a second
 			GetResult<std::string, u32, u8, u8>
-				result = result_queue.pop_front(1000);
+				result = result_queue.pop_front(4000);
```

Comment:
Why is it better to wait for 4s? Does not match the comment above (still says 1s)

Reply:
It started giving timeouts when the number of threads increased, and I did not get timeouts with 4 seconds, event with 8 mesh gen threads.

Fixed the comment.